vault/token_store.go

```diff
@@ -1818,10 +1818,11 @@ func (ts *TokenStore) handleTidy(ctx context.Context, req *logical.Request, data
 				}
 
 				var deletedChildrenCount int64
-				for _, child := range children {
+				for index, child := range children {
 					countParentList++
 					if countParentList%500 == 0 {
-						ts.logger.Info("checking validity of tokens in secondary index list", "progress", countParentList)
+						progress := float64(index)/float64(len(children))*100
+						ts.logger.Info("checking validity of tokens in secondary index list", "progress", progress)
```

Comment:
Instead of replacing the count value, can we include that in addition to the percentage value? 

```
ts.logger.Info("checking validity of tokens in secondary index list", "progress", countParentList, "percent_complete", percentComplete)
```

Reply:
Sure thing! I've pushed a new commit with the change